src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -17,6 +17,7 @@ type Foo implements Bar {
   four(argument: String = "string"): String
   five(argument: [String] = ["string", "string"]): String
   six(argument: InputType = {key: "value"}): Type
+  seven(argument: Int = null): Type
```

Comment:
Default value will be present inside `resolve`

Nullable type without value (and default value) will be omitted === `undefined`

[RFC](https://github.com/facebook/graphql/pull/83/files#diff-fe406b08746616e2f5f00909488cce66R746)